src/aims_ui/page_address_info.py

```diff
@@ -0,0 +1,34 @@
+import os
+from . import app
+from .models.get_endpoints import get_endpoints
+from .api_interaction import api
+from .models.get_addresses import get_addresses
+from flask import render_template
+from flask_login import login_required
+import json
+
+
+@login_required
+@app.route('/address_info/<uprn>')
+def address_info(uprn):
+  """Show all info about an address given the UPRN"""
+
+  result = api(
+      '/addresses/uprn/',
+      'uprn',
+      {'uprn': uprn},
+  )
+
+  if result.status_code == 200:
+    matched_addresses = get_addresses(result.json(), 'uprn', app)
```

Comment:
`app` is a global variable throughout the whole program - why is it being passed in here? Can't it be accessed from address.py as a global?

Reply:
I get  
 ```
File "./address-index-ui/src/aims_ui/models/get_addresses.py", line 11, in get_addresses
    addresses = [Address(address, app)]
NameError: name 'app' is not defined
```
When removing the parameter, is it possible it's only available when imported? There's several instances of 'app' being passed through for lots of the helper functions as they're unable to import from above their filepath location :/

Reply:
Fixed this using the `from aims_ui import app ` line and removing app as a passthrough parameter